lib/config.js

```diff
@@ -12,7 +12,11 @@ const appConfigPath = path.join(appDir, 'config.json')
 function getConfigFromFile () {
   const configFileExists = fse.existsSync(appConfigPath)
   if (configFileExists) {
-    return fse.readJsonSync(appConfigPath)
+    try {
+      return fse.readJsonSync(appConfigPath)
+    } catch (e) {
+      console.error(`Could not load config from ${appConfigPath}, please check your JSON is well formed.`)
```

Comment:
Not sure how this error message will read to someone who isn't a developer, @NoraGDS what do you think?

Reply:
Would you be happier if this said something like "`Could not load config from ${appConfigPath}, please check your configuration is formatted correctly.`"

Reply:
Or would you prefer I revert the catch here also?

Reply:
@lfdebrux doesn't ready too well. Can we say 'Check your configuration is formatted correctly in (whatever place it's in)'